src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+@import './utils/mixin';
+@import './utils/variable';
+@import './blocks/main';
+@import './blocks/card';
+@import './blocks/header';
+
+html {
+  font-family: 'Roboto ', sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing` property should be applied to all elements. It's recommended to use `box-sizing: border-box;` universally across all elements using the `*, *::before, *::after` selector.